src/base/static/components/organisms/dashboard/base-table.tsx

```diff
@@ -0,0 +1,266 @@
+/** @jsx jsx */
```

Comment:
Shouldn't this component be a molecule?

Nesting our components under this `/organisms/dashboard/` dir isn't consistent with our atomic design approach. If we are pushing the limits of atomic design, we should discuss whether we need a different approach.

Reply:
I'm revisiting the organization here. What about `BaseTable` makes it a molecule? I think the definition of an organism fits:

```
Organisms are relatively complex UI components composed of groups of molecules and/or atoms and/or other organisms. 
```

Reply:
I'd say things like table rows, cells, columns, etc. would be molecules, but it seems tedious to break those out into separate molecules files because they're only consumed in `BaseTable`.